src/style.scss

```diff
@@ -0,0 +1,10 @@
+@import 'https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap';
```

Comment:
Importing Google Fonts directly in SCSS is not recommended. It's better to include the font link in the HTML file to ensure fonts are loaded correctly and efficiently.